tests/test_integration_employee.py

```diff
@@ -9,14 +9,15 @@
 
 import freezegun
 
-from mora import lora
+from mora import lora, settings
+from mora.service import employee, orgunit
 from . import util
 
 mock_uuid = 'f494ad89-039d-478e-91f2-a63566554bd6'
 
 
 @freezegun.freeze_time('2017-01-01', tz_offset=1)
-@patch('mora.service.employee.uuid.uuid4', new=lambda: mock_uuid)
+@patch('mora.service.employee.uuid4', new=lambda: mock_uuid)
```

Comment:
`s/mora.service.employee.uuid4/uuid.uuid4/`

Since we're doing this all over the place, we should probably write a decorator that injects a simple, incrementing replacement for `uuid4()`.

Reply:
My absolutely biggest gripe with replacing it with a more generic patching, as mentioned earlier, is that we then mock _every_ use of `uuid` everywhere. In this case it doesn't make a difference, as nothing else depends on it, or at least not in a way that breaks stuff when we mock it out, but we never know.

Reply:
My point is that we _should_ patch `uuid4()` — `https://github.com/nicwest/notsouid` looks interesting…

Reply:
I initially just rolled my own, simplified decorator that does something similar to notsouid. I'll just replace it and be done with it.